vagrant/dev/bootstrap.sh

```diff
@@ -20,6 +20,7 @@ yum install -y \
 	docker \
 	python-devel python-tox \
 	python-virtualenv python-virtualenvwrapper python-pip \
-	libffi-devel \
+  enchant \
```

Comment:
To do this branch I needed to `pip install -e .[doc,dev]` (or at least, I did that) on the dev box. These dependencies were necessary.